src/neo-vm/StackItem.cs

```diff
@@ -26,6 +28,7 @@ public sealed override bool Equals(object obj)
         public static StackItem FromInterface<T>(T value)
             where T : class
         {
+            if (value is null) return Null;
```

Comment:
This could have been anything here... including `new ByteArray(null)` that flags internally as `null`. Another options is a `null` InteropInterface.

Reply:
`null` is null. It doesn't matter how to implement.

Reply:
Ok,so my opinion is to do it as a special case of ByteArray stack item. I think this simplifies more than having another stack item. Sorry for complaining so much here, but I think this is a very important thing (simplicity).

Reply:
And is not more simpler using a new StackItem?

Reply:
A new Stack Item cannot be simpler than having five others already... I thought you did that not for simplicity, but as a first solution to the problem. But if you guys find it more simple to have one more class, I'll understand.

Reply:
Shargon, I tried my approach, and now I agree it's not better than this one. ByteArray(null) caused several other tests to be required, specially when depending on Null/NotNull behavior. 
So, I reverted all my three last commits. Thanks for the time and discussion.